src/components/DocumentList/DocumentList.js

```diff
@@ -1,12 +1,14 @@
 import React from "react";
-import "font-awesome/css/font-awesome.min.css";
+import PropTypes from "prop-types";
+import { documentType } from "../../types";
```

Comment:
Calling a centralized propType. Since document is used by many components, it makes sense to declare once and reference.